ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/ViewScanConfigDetail.tsx

```diff
@@ -54,13 +56,20 @@ function ViewScanConfigDetail({
     isLoading,
     error = null,
 }: ViewScanConfigDetailProps): React.ReactElement {
+    const { scanConfigId } = useParams();
```

Comment:
Is there a benefit to using `scanConfigId` from params versus getting it from the `scanConfig` prop that's already being passed in?

Reply:
Yeah, so there's no big benefit other than being slightly cleaner. But that can be subjective. So either we do
```
const { scanConfigId } = useParams();
...
const lastSnapshot = complianceReportSnapshots[scanConfigId];
```
or
```
const lastSnapshot = complianceReportSnapshots[scanConfig?.id ?? ''];
```
I had initially done the second option. Then I checked what we did for VM Reports and we had done the first approach there. I thought it looked cleaner so I mimicked it here.

Reply:
Up to you all what you prefer. They both have the same outcome.